dotnet/src/webdriver/BiDi/Browser/BrowserModule.cs

```diff
@@ -96,4 +91,12 @@ protected override void Initialize(JsonSerializerOptions options)
 [JsonSerializable(typeof(GetClientWindowsResult))]
 [JsonSerializable(typeof(SetDownloadBehaviorCommand))]
 [JsonSerializable(typeof(SetDownloadBehaviorResult))]
+
+#pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
+[JsonSourceGenerationOptions(
+    PropertyNameCaseInsensitive = true,
```

Comment:
We are duplicating shared json options. Is it good? Or do we want to return back the construction of json context in runtime rather than generated options?